### DISCUSSION TITLE
My dear French friend, OpenVPN is ridiculously slow.

### DISCUSSION THREAD

**Discussion by dolbykoray** [angristan/openvpn-install#1261] [Help]
I really love you my friend, your work is great, greetings from Turkey. I installed the Ubuntu-OpenVPN version but there is a slowness in the system, I tried both with the configuration you suggested and manually, the problem did not go away. I am using speeds close to 300Mbit without VPN, but when I turn on VPN, the speed drops to 20Mbit. How can we solve this?

**Comment by dolbykoray** [comment]
Please help :'(

**Comment by gbnk0** [comment]
How is your server's cpu?
Regards

**Comment by dolbykoray** [comment]
quad core intel xeon cpu no problem 4gb ram 30gb m2ssd, tls very cpu using, how can encrtyp disable?

**Comment by gbnk0** [comment]
You mean CPU usage is high? How many clients have you?

**Comment by dolbykoray** [comment]
Thanks for your answer my friend. I can say that I have completely purified Ubuntu, there are only 2 different apps. nmon-openvpn- other apps may be Ubuntu system software, when I checked the cpu usage, openvpn uses cpu heavily, when I started speedtest, when I encrypted the packet numbers, I interpreted it as showing processor heavy usage.

**Comment by ruairinewman** [comment]
@dolbykoray This may or may not help you, but while running OpenVPN Community on a low-spec AWS instance, I found that the more secure, higher-bit ciphers were significantly more taxing, and resulted in noticeably slower throughput. Have you tried using a 128-bit cipher? While it's not ideal, it might at least narrow down what's causing your performance degradation.
